### ISSUE TITLE
fixing form URL for us-east-1

### ISSUE THREAD

**Issue by alexpcoleman** [eddturtle/direct-upload#1]
Hey there. First off, thanks for putting this together – it saved me a boatload of time :)

Just one thing I noted that needs to be fixed: I'm working in the "us-east-1" region, so I had to modify the `Signature::getFormUrl()` function to _not_ include the region when working in "us-east-1", since 
"us-east-1" URLs are simply `http://[bucket].s3.amazonaws.com/...`.

```
  return "//" . $this->bucket . "." . self::SERVICE .
            ($this->region->getName() != 'us-east-1' ? "-" . $this->region->getName() : '')
            . ".amazonaws.com";
```

**Comment by eddturtle** [comment]
Thanks for the feedback - should be fix now in 24dc5d35f4094e9538001b8eb3ba977502d0769d

**Comment by alexpcoleman** [comment]
Great, thanks! :)
